tests/pyccel/project_class_imports/project/shapes/Square_mod.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring, relative-beyond-top-level
+from ..basics.Point_mod import Point
+
+class Square:
+    def __init__(self, a : Point, b : Point, c : Point, d : Point):
+        self.a = a
+        self.b = b
+        self.c = c
+        self.d = d
```

Comment:
How can you be sure that this shape is a square? 😜

Reply:
:sweat_smile: